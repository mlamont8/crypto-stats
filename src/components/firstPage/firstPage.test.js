import React from 'react';
import renderer from 'react-test-renderer';
import FirstPage from './firstPage';

describe('First Page component', () => {
    it('renders correctly', () => {
        const renderedImage = renderer.create(
            <FirstPage />
        );
        expect(renderedImage.toJSON()).toMatchSnapshot();
    })
})