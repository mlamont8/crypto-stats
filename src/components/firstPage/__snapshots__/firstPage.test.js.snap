// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`First Page component renders correctly 1`] = `
<div
  className="container-fluid firstPage"
>
  <span>
     FIRST PAGE 
  </span>
  <span>
     First render 
  </span>
</div>
`;
